code-challenges.js

```diff
@@ -3,8 +3,23 @@
 
 // --------------------1) Create a function that returns the first 10 numbers of the Fibonacci sequence in an array. Expected output: [1, 1, 2, 3, 5, 8, 13, 21, 34, 55]
 
-
-
+// function fibonacci makes 3 variables that are put into an equation and return an array with 10 indexs
+//can I use state? should state hold "n" and refer to it and use it as part of the equatiopn updating it each time?
+//for loops i guess --- try to find map or filter soulution if time permits
+var x = 1
+var n = 1
+const fibb = () => {
+  let fibbArray = []
+  // push is higherorder function
+  fibbArray.push(x)
+  fibbArray.push(n)
+  for (var i = 2; i < 10; i++) {
+    fibbArray.push(fibbArray[i - 2] + fibbArray[i -1])
+  }
+return fibbArray
+}
```

Comment:
good solution! you could probably omit your x and n variables though and just push 2 1's into your array. it might reflect your intent better?